201-web-app-with-app-gateway-v2/azuredeploy.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": { 
+        "webAppName": {
```

Comment:
remove these since you have defaultValues

Reply:
done